trymito.io/pages/teams.tsx

```diff
@@ -0,0 +1,232 @@
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import Image from 'next/image'
+import Footer from '../components/Footer/Footer';
+import Header from '../components/Header/Header';
+import securityStyles from '../styles/Security.module.css'
+import pageStyles from '../styles/Page.module.css'
+import titleStyles from '../styles/Title.module.css'
+import homeStyles from '../styles/Home.module.css'
+import teamsStyles from '../styles/Teams.module.css'
+import CTAButtons from '../components/CTAButtons/CTAButtons';
+import ContactCTACard from '../components/CTACards/ContactCTACard';
+
+// Import Icons
+import JupyterIcon from '../public/icon-squares/JupyterIcon.svg'
+import SupportIcon from '../public/icon-squares/SupportIcon.svg'
+import DocsIcon from '../public/icon-squares/DocsIcon.svg'
+import PrivateIcon from '../public/icon-squares/PrivateIcon.svg'
+import Link from 'next/link';
+
+
+const Teams: NextPage = () => {
+
+  return (
+    <>
+      <Head>
+        <title>Mito | Teams </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+      </Head>
+      
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+
+            <section className={titleStyles.title_card + ' ' + titleStyles.grid_card}>
+                <h1 className={titleStyles.title}>
+                    Embrace Python by leveraging your team’s Excel mastery
+                </h1>
+                <p className={titleStyles.description}>
+                    Onboard your team to Mito in minutes
+                </p>
+                <div className={homeStyles.cta_button_and_video_spacer}>
+                    <CTAButtons variant='contact' />
+                </div>
+            </section>
+            <section>
+                <div className={pageStyles.subsection}>
+                    <div className={homeStyles.functionality_text}>
+                        <p>
+                            <b className='text-primary'>Spreadsheet-first teams </b> use Mito to transition to Python while preserving 
```

Comment:
This has a some filler words methinks - and also "we" are the least relevant mfs here. Maybe something like:

- Spreadsheet-first teams use Mito to write Python from the spreadsheet interface they know and love. (we 
- Mito helps **spreadsheet-first** teams transition their spreadsheet expertise to write Python code. (this one also puts Mito first rather than the teams, which might be better?)